### PR TITLE
Retail upgradepath more 41 keichwa

### PR DESCRIPTION
**Pull request by keichwa**
- New: short statement about branchserver upgrade -- @aaannz , do we need more?
- Minor cleanup edits

It also belongs to https://github.com/SUSE/spacewalk/issues/10774.

### CODE REVIEW

**Comment by keichwa** [comment]
@aaannz I forgot to mention that I changed the repository names for SP2 and 4.1.  I was not able to verify it.  Please, take a closer look!

File: modules/retail/pages/retail-upgrade-branchserver.adoc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[[retail-upgrade-branchserver]]
+= Upgrade {smr} Branch Server
+
+This section describes upgrading a {smr} Branch Server to the next SP (service pack).
+
+{smr} Branch Server is a client system such as a {productname} Proxy with additional {smr} features.
+For more information about upgrading a proxy client, see xref:upgrade:proxy-update.adoc[].
```
Comment (aaannz) [inline modules/retail/pages/retail-upgrade-branchserver.adoc]: A note that `Highstate should be applied on migrated branch/proxy to update retail functionality` should be here.
Just updating packages will not update retail configurations (i.e. new http mode will not be enabled by package update). Assuming that suma server is updated first.

Comment (keichwa) [reply modules/retail/pages/retail-upgrade-branchserver.adoc]: ok, thanks.  I hope I got it right.

File: modules/retail/pages/retail-upgrade-branchserver.adoc [hunk 2]
```diff
@@ -4,4 +4,15 @@
 This section describes upgrading a {smr} Branch Server to the next SP (service pack).
 
 {smr} Branch Server is a client system such as a {productname} Proxy with additional {smr} features.
-For more information about upgrading a proxy client, see xref:upgrade:proxy-update.adoc[].
+
+[NOTE]
+====
+Upgrade the {productname} Server before beginning with the {srm} upgrade.
```
Comment (Loquacity) [inline modules/retail/pages/retail-upgrade-branchserver.adoc]:
```suggestion
Upgrade the {productname} Server before starting the {smr} upgrade.
```

File: modules/retail/pages/retail-upgrade-branchserver.adoc [hunk 3]
```diff
@@ -4,4 +4,15 @@
...
+====
+
+
```
Comment (Loquacity) [inline modules/retail/pages/retail-upgrade-branchserver.adoc]:
```suggestion
```

File: modules/retail/pages/retail-upgrade-branchserver.adoc [hunk 4]
```diff
@@ -4,4 +4,15 @@
...
+
+.Procedure: Upgrading the {smr} Branch Server
+. For general information about upgrading a proxy client, see xref:upgrade:proxy-update.adoc[].
+. On the {smr} Branch Server apply highstate after the general proxy upgrade finished successfully.
```
Comment (Loquacity) [inline modules/retail/pages/retail-upgrade-branchserver.adoc]:
```suggestion
. After the proxy upgrade is complete, apply the highstate on the {smr} Branch Server.
```

File: modules/retail/pages/retail-upgrade-branchserver.adoc (hunk 3 above)
Comment (keichwa) [reply modules/retail/pages/retail-upgrade-branchserver.adoc]: I think we want 3 empty lines before a procedure title ;)

**Comment by keichwa** [comment]
thanks for all your help, Ondrej and Lana!

**Pull request merged**